File: packages/transform-io/typescript/src/extension-to-transform-io.ts
const extensionToTransformIo = new Map([
  ["h5", "hdf5"],
  ["hdf5", "hdf5"],
  ["txt", "txt"],
  ["mat", "mat"],
  ["xfm", "mnc"],
  ["iwt", "wasm"],
  ["iwt.cbor", "wasm"],
  ["iwt.cbor.zst", "wasm-zstd"],
]);

export default extensionToTransformIo;

File: packages/transform-io/python/itkwasm-transform-io-wasi/itkwasm_transform_io_wasi/extension_to_transform_io.py
from collections import OrderedDict

extension_to_transform_io = OrderedDict([
  ("h5", "hdf5"),
  ("hdf5", "hdf5"),
  ("txt", "txt"),
  ("mat", "mat"),
  ("xfm", "mnc"),
  ("iwt", "wasm"),
  ("iwt.cbor", "wasm"),
  ("iwt.cbor.zst", "wasm-zstd"),
])
